packages/blob/src/client.ts

```diff
@@ -46,12 +46,34 @@ export const put = createPutMethod<ClientPutCommandOptions>({
 // upload()
 // This is a client-side wrapper that will fetch the client token for you and then upload the file
 export interface UploadOptions {
+  /**
+   * Whether the blob should be publicly accessible. Support for private blobs is planned.
+   */
   access: 'public';
+  /**
+   * Defines the media type of the blob. By default, this value is inferred from the pathname.
```

Comment:
```suggestion
   * Defines the media type of the blob. By default, this value is inferred from the pathname. Sent as the 'content-type' header when downloading a blog
```

Not sure we should be that specific, and if so it should also map what we have on the vercel.com documentation I guess.